SIL.Core/Xml/XmlSerializationHelper.cs

```diff
@@ -139,27 +139,35 @@ public static byte[] SerializeToByteArray<T>(T data, bool omitXmlDeclaration)
 		/// Serializes an object to an XML string.
 		/// </summary>
 		/// ------------------------------------------------------------------------------------
-		public static string SerializeToString<T>(T data)
+		public static string SerializeToString<T>(T data, Encoding encoding = null)
```

Comment:
This is a breaking change, i.e. a client needs to be recompiled in order to work with this API. This means that the commit message should have a `+semver:major` added.

Or we make it a real overload `public static string SerializeToString<T>(T data, Encoding encoding)` and add `+semver:minor` to the commit message. Same for the other method below.

Depending on what you decide to do the line in the changelog might have to be moved to the `Changed` section.

Reply:
I think having a default value is the more elegant solution and since we have an unreleased major version change I'd go with `+semver:major`